scripts/rv_expression_association/debug_maf_distributions.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+import hail as hl
+# from bokeh.io import show
+from bokeh.io.export import get_screenshot_as_png
+from cpg_utils.hail_batch import output_path, init_batch 
+
+init_batch()
+mt = hl.read_matrix_table('gs://cpg-tob-wgs-test/tob_wgs_vep/v1/vep105_GRCh38.mt')
+mt = hl.experimental.densify(mt)
+mt = hl.variant_qc(mt)
+mt = hl.filter_intervals(
+    mt, [hl.parse_locus_interval('chr22:23704425-23802743', reference_genome='GRCh38')]
+)
+mt = mt.filter_rows(hl.len(hl.or_else(mt.filters, hl.empty_set(hl.tstr))) == 0)
```

Comment:
It would be good to add a `checkpoint` after this line, as otherwise all operations above will be reexecuted for line 23 and after.

Reply:
i've tried adding a checkpoint (in the `get_rv_outlier.py` script) at lines 85-86 but not super sure of the syntax!